templates/zerver/help/alert-words.md

```diff
@@ -0,0 +1,26 @@
+# Alert Words
+Zulip allows you to define custom alert words and phrases, which when said will notify you, similarly to being [mentioned](/help/at-mention-a-team-member) by another user.
```

Comment:
`which when said will notify you` is a bit unclear. I'd stop the sentence at `phrases` and start a new one with `When they are used in the streams you are subscribed to, it works similarly to being [@-mentioned](link) by another user.`